screengrab/lib/screengrab/runner.rb

```diff
@@ -136,9 +136,10 @@ def screenshot_file_names_in(output_directory, device_type)
     end
 
     def determine_external_screenshots_path(device_serial)
-      device_ext_storage = run_adb_command("adb -s #{device_serial} shell echo \\$EXTERNAL_STORAGE",
+      device_ext_storage = run_adb_command("adb -s #{device_serial} shell echo \$EXTERNAL_STORAGE",
```

Comment:
Commented this on the other PR that this is based off of but just wanted to put this here to so that this doens't accidentally get merged in but `\\` is needed on mac 😊 

(This will get addressed once rebased after other PR gets merged in to master)